charts/incubator/invidious/values.yaml

```diff
@@ -0,0 +1,158 @@
+image:
+  repository: tccr.io/truecharts/invidious
+  pullPolicy: IfNotPresent
+  tag: latest@sha256:bb8e2e0eedf9296a1af274e09d898555b0bfbd62aec52a346bddfb9b3b1dc983
+
+invidious:
+  db_name: "{{ .Values.postgresql.postgresqlDatabase }}"
+  db_user: "{{ .Values.postgresql.postgresqlUsername }}"
+  db_pass:
+    secretKeyRef:
+      name: dbcreds
+      key: postgresql-password
+  db_host:
+    secretKeyRef:
+      name: dbcreds
+      key: plainhost
+  db_port: 5432
+  check_tables: true
```

Comment:
Things that won't be exposed to users to edit, can be directly defined in the configmap